nipype/pipeline/plugins/multiproc.py

```diff
@@ -251,14 +218,15 @@ def _send_procs_to_workers(self, updatehash=False, graph=None):
                         key=lambda item: (self.procs[item]._interface.estimated_memory_gb,
                                           self.procs[item]._interface.num_threads))
 
-        if str2bool(config.get('execution', 'profile_runtime')):
+        resource_monitor = str2bool(config.get('execution', 'resource_monitor', 'false'))
```

Comment:
May want to check for `profile_runtime`, use value and raise deprecation warning.

Reply:
Oh yeah

Reply:
Added a deprecation on the config itself